### DISCUSSION TITLE
How to specify changes of a single file, between 2 distinct repositories, in a pull request?

### DISCUSSION THREAD

**Discussion by ZjYwMj** [community/community#46659] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

I copied a single file from [upstream repository](https://github.com/torvalds/linux/) to [my newly created for this task repository](https://github.com/ZjYwMj/linux). I than changed the file I copied. Now I want to create a pull request. I am asked about comparing changes. How do I specify [upstream file](https://github.com/torvalds/linux/tree/master/kernel/power/Kconfig) ... [file modified by me](https://github.com/ZjYwMj/linux/tree/main/kernel/power/Kconfig) ?

**Comment by airtower-luna** [comment, marked as answer]
You can't, because your repository is not a *fork* of the upstream one. To send a pull request you need to fork the upstream repository, make your edit there, and then create a PR. See the [Collaborative development](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models) section of the documentation for how to do it.

However, the Linux kernel isn't developed via PRs on GitHub, so making one isn't gonna help you. Patches for Linux must be submitted via the right mailing list, the process is described in the kernel documentation: https://github.com/torvalds/linux/blob/master/Documentation/process/submitting-patches.rst
